Project.toml

```diff
@@ -10,13 +10,12 @@ Compose = "a81c6b42-2e10-5240-aca2-a61377ecd94b"
 Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
-DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
-Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
 GraphPlot = "a2cc645c-3eea-5389-862e-a155d0052231"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
```

Comment:
b6ca364a60cb8c93cfe767b43b32abe6de40a3a1 comments out the `@reexports`. Leaving this comment here as a reminder to remove it as a dependency before merging.